common/utils/src/main/resources/error/error-classes.json

```diff
@@ -149,6 +149,19 @@
     ],
     "sqlState" : "42846"
   },
+  "CANNOT_CREATE_DATA_SOURCE_V2_TABLE" : {
```

Comment:
I can't find other errors that mention data source v2. I think it's a developer thing and we should not expose it to end users via error message. How about just `CANNOT_CREATE_TABLE`?